src/faebryk/exporters/visualize/interactive_graph.py

```diff
@@ -8,10 +8,16 @@
 from faebryk.core.graphinterface import GraphInterface
 from faebryk.core.link import Link
 from faebryk.core.node import Node
-from faebryk.exporters.visualize.util import IDSet, generate_pastel_palette
+from faebryk.exporters.visualize.util import (
+    generate_pastel_palette,
+    offer_missing_install,
+)
+from faebryk.libs.util import FuncSet
 
 
 def interactive_graph(G: Graph):
+    offer_missing_install("dash_cytoscape")
+    offer_missing_install("dash")
```

Comment:
See comment in visualize.
Having a poetry dep group for this and then check if its there and if not print: install via `poetry ...` might be cleaner

Reply:
Yeah, this one's a little weird. Not sure I love the `offer_missing` pattern for the deps at all.

Separately, we should shoot for a vanilla `pip`/`pipx` install working by default, not `poetry`. Let's detangle that where when we come by it

Reply:
Afaik currently everything works with a pip only setup